keep/providers/opsgenie_provider/opsgenie_provider.py

```diff
@@ -30,13 +30,37 @@ class OpsGenieRecipient(pydantic.BaseModel):
 class OpsgenieProvider(BaseProvider):
     """Create incidents in OpsGenie."""
 
+    PROVIDER_SCOPES = [
+        ProviderScope(
+            name="opsgenie:create",
+            description="Create OpsGenie alerts",
+            mandatory=True,
+            alias="Create alerts",
+        ),
+    ]
+
     def __init__(
         self, context_manager: ContextManager, provider_id: str, config: ProviderConfig
     ):
         super().__init__(context_manager, provider_id, config)
         self.configuration = opsgenie_sdk.Configuration()
         self.configuration.api_key["Authorization"] = self.authentication_config.api_key
-
+        
+    def validate_scopes(self):
+        scopes = {}
+        self.logger.info("Validating scopes")
+        try:
+            self._create_alert(
```

Comment:
@ezhil56x yo folk, the problem is that we can't just "create alert" on users' OpsGenie, we need to somehow validate it without actually changing the data. I would look for some API that returns the scopes for the creds.

Reply:
Missed that will work on it

Reply:
> @ezhil56x yo folk, the problem is that we can't just "create alert" on users' OpsGenie, we need to somehow validate it without actually changing the data. I would look for some API that returns the scopes for the creds.

For now only Creating Alert is supported on keep right?

Reply:
Yes